gensim/models/ldamodel.py

```diff
@@ -823,72 +830,32 @@ def get_topic_terms(self, topicid, topn=10):
         Only return 2-tuples for the topn most probable words (ignore the rest).
 
         """
-        topic = self.state.get_lambda()[topicid]
+        topic = self.get_topics()[topicid]
         topic = topic / topic.sum()  # normalize to probability distribution
         bestn = matutils.argsort(topic, topn, reverse=True)
         return [(id, topic[id]) for id in bestn]
 
-    def top_topics(self, corpus, num_words=20):
+    def top_topics(self, corpus=None, texts=None, dictionary=None, window_size=None,
+                   coherence='u_mass', topn=20, processes=-1):
```

Comment:
Several papers have shown that sliding-window based coherence metrics have higher correlation with human judgements, and `u_mass` has lower correlations with human judgements than these methods. Why limit to `u_mass` when it is not the best technique for coherence calculations?